### PR TITLE
Feat/quota

### PR DESCRIPTION
**Pull request by skps2010**

### CODE REVIEW

File: model/problem.py [hunk 1]
```diff
@@ -59,6 +59,9 @@ def view_problem_list(user, offset, count, problem_id, tags, name, course):
                     'submitter': p.submitter,
                     'tags': p.tags,
                     'type': p.problem_type,
+                    'quota': p.quota,
+                    'submitCount': user.problem_submission.get(
```
Comment (as535364) [inline model/problem.py]: submitCount for what?

File: model/submission.py [hunk 2]
```diff
@@ -96,6 +96,17 @@ def create_submission(user, language_type, problem_id):
                 'got': language_type
             },
         )
+    # reset quota if it's a new day
+    if user.last_submit.date() != now.date():
```
Comment (as535364) [inline model/submission.py]: if student didn't submit any submission?

File: model/problem.py (hunk 1 above)
Comment (skps2010) [reply model/problem.py]: a player's submission count for that problem in that day

File: model/submission.py (hunk 2 above)
Comment (skps2010) [reply model/submission.py]: then he can submit?

Comment (skps2010) [reply model/submission.py]: last_submit is a very early day by default

Comment (as535364) [reply model/submission.py]: ok

File: model/problem.py (hunk 1 above)
Comment (as535364) [reply model/problem.py]: so, all the information is stroed in problem?

Comment (skps2010) [reply model/problem.py]: no, submission count is stored in User

**Pull request merged**